catalog-item.html

```diff
@@ -1,15 +1,333 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
-</head>
-
-<body>
-
-    
-</body>
-
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Details</title>
+</head>
+
+<body>
+    <header class="page-header">
+        <a href="#" class="main-logo"><img src="#" alt="pixmart logo">Pixmart.</a>
+        <nav class="page-nav">
+            <ul>
+                <li class="nav-item"><a href="#">About</a></li>
+                <li class="nav-item"><a href="#">Products</a></li>
+                <li class="nav-item"><a href="#">Creators</a>
+                    <p class="nav-item-tag">NEW</p>
+                </li>
+                <li class="nav-item"><a href="#">Blog</a></li>
+                <li class="nav-item"><a href="#">Contact</a></li>
+            </ul>
+            <div class="nav-buttons">
+                <button type="button" class="login-button">
+                    <img src="#" alt="login-logo"></button>
+                <button type="button" class="cart-button">
+                    <img src="#" alt="cart-logo"></button>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="item-information">
+            <div class="gallery">
+                <div class="gallery-item">
+                    <img class="item-image-large" src="#" alt="item image">
+                    <button type="button" class="expand-button">&#x2922;</button>
+                </div>
+                <div class="gallery-item-selection">
+                    <ul class="gallery-item-list">
+                        <li class="list-item"><img class="item-image" src="#" alt="item image"></li>
+                        <li class="list-item"><img class="item-image" src="#" alt="item image"></li>
+                        <li class="list-item"><img class="item-image" src="#" alt="item image"></li>
+                        <li class="list-item"><img class="item-image" src="#" alt="item image"></li>
+                    </ul>
+                    <button class="previous-item">&#8249;</button>
+                    <button class="next-item">&#8250;</button>
+                </div>
+            </div>
+            <div class="main-content">
+                <div class="top-content">
+                    <img src="#" alt="rating star">
+                    <img src="#" alt="rating star">
+                    <img src="#" alt="rating star">
+                    <img src="#" alt="rating star">
+                    <img src="#" alt="rating star">
+                    <p class="rating-text">(50)</p>
+                    <!-- I was also wondering if this could be wrapped in figure:? -->
```

Comment:
we will talk about "stars" element a little bit later 
now you can just group starts and rating-text in common container